apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css

```
@@ -0,0 +1,131 @@
+.sign-in__container {
+	position: relative;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	gap: 30px;
+	height: 100dvh;
+	padding: 15px;
+	overflow: hidden;
+	background-color: var(--background);
+}
+
+.sign-in__form-container {
```

```suggestion
.form-container {
```
These styles are only for sign-in screen, no need to specify sign-in each time. Same for all